ci/pipeline.yml

```diff
@@ -1,179 +1,234 @@
       - do:
-        - task: push-tiles
-          file: omg-src-in/ci/tasks/push-tiles.yml
-          params:
-            env_file_name: {{env_file_name}}
-          on_failure: &destroy_infrastructure
-            task: destroy-infrastructure
-            file: omg-src-in/ci/tasks/destroy-infrastructure.yml
+          - task: push-tiles
+            file: omg-src-in/ci/tasks/push-tiles.yml
             params:
-              google_project: {{google_project}}
-              google_json_key_data: {{google_json_key_data}}
-              env_file_name: {{env_file_name}}
-        - task: deploy-pcf
-          file: omg-src-in/ci/tasks/deploy-pcf.yml
-          params:
-            env_file_name: {{env_file_name}}
-          on_failure:
-            do:
-            - &destroy_pcf
-              task: destroy-pcf
-              file: omg-src-in/ci/tasks/destroy-pcf.yml
-              params:
-                google_project: {{google_project}}
-                google_json_key_data: {{google_json_key_data}}
-                env_file_name: {{env_file_name}}
-            - *destroy_infrastructure
+              env_file_name: ((env_file_name))
+          - task: deploy-pcf
+            file: omg-src-in/ci/tasks/deploy-pcf.yml
+            params:
+              env_file_name: ((env_file_name))
 
   - name: run-certification
     serial: true
     serial_groups: [gcp-project]
     plan:
       - aggregate:
-        - {trigger: true, passed: [deploy-pcf], get: omg-src-in, resource: omg-src-in}
-        - {trigger: true, passed: [deploy-pcf], get: omg-env, resource: omg-env}
+          - trigger: true
+            passed: [deploy-pcf]
+            get: omg-src-in
+            resource: omg-src-in
+          - trigger: true
+            passed: [deploy-pcf]
+            get: omg-env
+            resource: omg-env
       - aggregate:
         task: run-certification
         file: omg-src-in/ci/tasks/run-certification.yml
         params:
-          env_file_name: {{env_file_name}}
-        on_failure:
-          do: &destroy_all
-           - *destroy_pcf
-           - *destroy_infrastructure
+          env_file_name: ((env_file_name))
 
   - name: destroy-all
     serial: true
     serial_groups: [gcp-project]
     plan:
       - aggregate:
-        - {trigger: true, passed: [run-certification], get: omg-src-in, resource: omg-src-in}
-        - {trigger: true, passed: [run-certification], get: omg-env, resource: omg-env}
-      - do: *destroy_all
-        on_failure:
-          do: *destroy_all
+          - trigger: true
+            passed: [run-certification]
+            get: omg-src-in
+            resource: omg-src-in
+          - trigger: true
+            passed: [run-certification]
+            get: omg-env
+            resource: omg-env
+      - do: &destroy_all
+          - task: destroy-pcf
+            file: omg-src-in/ci/tasks/destroy-pcf.yml
+            params:
+              google_project: ((google_project))
```

Comment:
I don't think these are used in the destroy-pcf.yml, will you check that they're needed either remove them here or add them there?


I see the following errors when the pipeline starts:

```
[WARNING] env_file_name was defined in pipeline but missing from task file
[WARNING] google_json_key_data was defined in pipeline but missing from task file
[WARNING] google_project was defined in pipeline but missing from task file
```

Reply:
@josephlewis42 They are needed. Those warnings mean they're not defined in the task.yml they're used in. The pipeline vars are supposed to be overrides.

To prevent these warnings from happening, I could add 

```
params:
  a_param: replaceme
  b_param: replaceme
```

I've opened up an issue for this: #32